### PR TITLE
docs(dropdown): fix options doc blocks

### PR DESCRIPTION
**Pull request by manniL**
rename checkbox to dropdown

### CODE REVIEW

File: src/components/Dropdown.react.js [hunk 1]
```diff
@@ -111,19 +111,19 @@ Dropdown.propTypes = {
...
-             * The value of the checkbox. This value
+             * The value of the dropdown. This value
              * corresponds to the items specified in the
              * `values` property.
              */
             value: PropTypes.string,
 
             /**
-             * If true, this checkbox is disabled and can't be clicked on.
+             * If true, this dropdown is disabled and items can't be seleted.
```
Comment (T4rk1n) [inline src/components/Dropdown.react.js]: selected.

Comment (manniL) [reply src/components/Dropdown.react.js]: Whoops, 1 sec

**Comment by T4rk1n** [comment]
Great, thanks.

**Comment by T4rk1n** [comment]
@valentijnnieman I tagged you because some of those PR's needs a release after merge. I don't know how you want to proceed in those cases. I can merge them and do a PR for the changelogs and release if you want to.

**Comment by valentijnnieman** [comment]
@T4rk1n That sounds good to me!

**Pull request merged**